aws_lambda_powertools/metrics/__init__.py

```diff
@@ -20,4 +21,6 @@
     "MetricResolutionError",
     "SchemaValidationError",
     "MetricValueError",
+    "EMFProvider",
```

Comment:
I'm wondering if `CloudWatchProvider` could be a better name for this. In the end we will always use the name of the provider and not the output/parser format. Examples:
`DataDogProvider`
`OtherProvider`

it makes sense?

Reply:
Thanks for changing the name of the class @roger-zhangg! I realized that `CloudWatchEMFProvider` could be an even better name, as we can add a provider called `CloudWatchProvider` that sends metrics through the SDK, for example.